services/postgres/src/rotator.ts

```diff
@@ -0,0 +1,194 @@
+import { DescribeSecretCommandOutput } from '@aws-sdk/client-secrets-manager'
```

Comment:
Assuming this is based on the aws lambda rotator script we removed - can we add that in here in the comments somewhere in case someone wants to see where the approach was developed

Reply:
This comments also seems helpful to include in our version too for someone wondering whats happening

   _This script enable single-user rotation scheme to rotate an RDS PostgreSQL user credential. This rotation
    scheme logs into the database as the user and rotates the user's own password, immediately invalidating the
    user's previous password._

Reply:
Yup absolutely, I'll add some documentation like this into the script